pinot-query-runtime/src/test/resources/queries/BooleanLogic.json

```diff
@@ -0,0 +1,106 @@
+{
+  "boolean_type": {
```

Comment:
Can we add Select true from tbl, select xx from tbl where true (where col = true)